bcrs-ui/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --testPathPattern"
+  },
+  "dependencies": {
+    "@mdi/font": "^4.5.95",
+    "axios": "^0.18.0",
+    "core-js": "^3.4.3",
+    "sbc-common-components": "2.0.22",
+    "vue": "^2.6.10",
+    "vue-class-component": "^7.0.2",
+    "vue-property-decorator": "^8.3.0",
+    "vue-router": "^3.1.3",
+    "vuetify": "^2.0.11",
+    "vuex": "^3.1.2",
+    "vuex-class": "^0.3.2"
```

Comment:
What's this one for? Should we be using it in coops-ui in lieu of something else?

Reply:
This module provides a shortcut to binding store to our classes. Like the `@Props` decorator used by `Vue-class-component` it allows us to use `@State` , '@Actions` and so on. Proves a consistent looking class component :)  As far as updated Coops-ui, i'm not so sure. That would involve rewriting a lot of the store of as well. If you feel it's something we would be given time to do, i'm certain it could be done, but would be a fairly large task affecting many components ( everywhere store is used ).

Reply:
I've created #2179 to possibly implement this change in coops-ui.